src/app.jsx

```diff
@@ -31,19 +32,22 @@ const App = () => {
   }, [loggedIn]);
 
   return (
-    <div>
-      <Routes>
-        <Route path="/login" element={<LandingPage />} />
-        <Route path="/register" element={<LandingPage />} />
-        <Route element={<AppLayout />}>
-          <Route path="/dashboard" element={<Dashboard />} />
-          <Route path="/reserve-form" element={<ReserveForm />} />
-          <Route path="/my-reservations" element={<MyReservations />} />
-          <Route path="/add-car-item" element={<AddCarItem />} />
-          <Route path="/delete-car-item" element={<DeleteCarItem />} />
+    <Routes>
+      <Route path="/login" element={<LandingPage />} />
+      <Route path="/register" element={<LandingPage />} />
+
+      <Route element={<AppLayout />}>
+        <Route path="/dashboard" element={<Dashboard />} />
+        <Route path="/reserve-form" element={<ReserveForm />} />
+        <Route path="/my-reservations" element={<MyReservations />} />
+        <Route path="/add-car-item" element={<AddCarItem />} />
+        <Route path="/delete-car-item" element={<DeleteCarItem />} />
+
+        <Route path="/items" element={<Outlet />}>
+          <Route path=":id" element={<CarDetail />} />
         </Route>
-      </Routes>
```

Comment:
I suggest you replace this code above with this:  "<Route path="/items/:id" element={<CarDetail />} />"